client/src/pages/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useState } from 'react';
+import {
+  Button,
+  CssBaseline,
+  TextField,
+  Grid,
+  FormControl,
+  MenuItem,
+  Select,
+  Typography,
+  Container,
+  Paper,
+  Box,
+} from '@material-ui/core';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './useStyles';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+
+export default function EditProfile(): JSX.Element {
+  const classes = useStyles();
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [gender, setGender] = useState('male');
+  const optionSelect = [];
+  for (let i = 1; i < 32; i++) {
+    optionSelect.push(
+      <MenuItem value={i} key={i}>
+        {i}
+      </MenuItem>,
+    );
+  }
+
+  return (
+    <Container component="main" className={classes.main}>
+      <CssBaseline />
+      <Grid container>
+        <Grid item md={2} sm={12}>
```

Comment:
Eventually we will be creating a separate functional sidebar (part of another ticket). You can leave this in for the time being though :)